eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -153,13 +164,17 @@ public void exec() throws IOException {
     /**
      * Construct severities to fail on.
      * @param warnfail Fail on warning flag
+     * @param errfail Fail on error flag
      * @return Constructed set
      */
-    private static Set<String> failures(final boolean warnfail) {
+    private static Set<String> failures(final boolean warnfail, final boolean errfail) {
         final Set<String> failures = new HashSet<>(3);
         if (warnfail) {
             failures.add(Sanitized.WARNING);
         }
+        if (errfail) {
+            failures.add(Sanitized.ERROR);
+        }
```

Comment:
@Graur to me this design looks overly complicated. Can't we just check twice in the method above and delete this method?